src/data/stripe.ts

```diff
@@ -23,22 +19,34 @@ export const getDonationOptions = async () => {
 		const stripe = new Stripe(STRIPE_SK, {
 			apiVersion: '2022-08-01',
 		});
-		const { data: products = [] } = await stripe.products.list({
-			expand: ['data.default_price'],
+
+		const { data: paymentLinks = [] } = await stripe.paymentLinks.list({
+			active: true,
+			expand: ['data.line_items'],
 		});
 
-		const _options = products.map((option) => {
-			const default_price = option.default_price as FormattedPrice;
-			(option as unknown as FormattedProduct).price = {
-				id: default_price?.id,
-				formatted_amount: String(default_price.unit_amount / 100),
+		const _options = new Map();
+
+		for (const link of paymentLinks) {
+			const { id, line_items, url } = link;
+			const lineItem = line_items?.data[0] || {
+				amount_total: 0,
+				description: '',
 			};
+			const { amount_total, description } = lineItem;
 
-			return option as unknown as FormattedProduct;
-		});
+			if (!_options.has(description)) {
+				_options.set(description, {
+					id,
+					url,
+					amount: amount_total / 100,
+					name: description,
+				});
+			}
+		}
```

Comment:
🔍 The `paymentLinks` collection for some reason has some duplicate links, at least in the test environment. This can probably be fixed on the Stripe dashboard. Until then, we deduplicate each donation option by its name.